data.json

```diff
@@ -1738,6 +1886,29 @@
     "chains": "bsc",
     "tags": "Defi"
   },
+  {
+    "category": "Wallets",
+    "groupTitle": "",
+    "name": "iMe Messenger and Crypto Wallet",
+    "desc": "iMe is a messaging smart platform powered by Telegram with Crypto Wallet.",
+    "website": "https://www.imem.app/",
+    "isSupportStaking": true,
+    "chains": "bsc",
+    "tags": "Defi"
+  },
+  {
+    "category": "Wallets",
+    "groupTitle": "",
+    "name": "Grindery Wallet",
+    "logo": "https://reputation-bnb.anima.io/img/anima-white-icon.svg",
+    "desc": "Grindery Wallet integrates with Telegram making crypto transactions as simple as sending a message.",
+    "website": "https://www.grindery.com/",
+    "isSupportStaking": false,
+    "chains": "bsc,opbnb",
+    "tags": "Defi"
+  },
+https://https://www.grindery.com//
```

Comment:
not valid json format, please delete this line

Reply:
line 1910 deleted